dev/top_printers.ml

```diff
@@ -149,8 +147,8 @@ let genppj f j = let (c,t) = f j in (c ++ str " : " ++ t)
 
 let ppj j = pp (genppj (envpp pr_ljudge_env) j)
 
-let prsubst s = pp (Mod_subst.debug_pr_subst s)
-let prdelta s = pp (Mod_subst.debug_pr_delta s)
+let ppsubst s = pp (Mod_subst.debug_pr_subst s)
+let ppdelta s = pp (Mod_subst.debug_pr_delta s)
```

Comment:
Is `pp` for pretty-printer and `pr` for just printer? And is this your reason for changing a few names? @herbelin seems to having understood it ("uniformization of naming") but I don't.

Reply:
`pp` seems to be used for `'a -> unit`, `pr` for `'a -> Pp.t`.

Reply:
Oh! Thanks for the info!